cli/tools/doc.rs

```diff
@@ -87,7 +88,11 @@ pub async fn print_docs(
   };
 
   if doc_flags.json {
-    write_json_to_stdout(&doc_nodes)
+    let json_output = serde_json::json!({
+      "version": version::deno(),
```

Comment:
```suggestion
      "version": 1,
```

So not the deno version, but a version that signifies the schema version of this json. Then we can bump this number whenever it changes. So we can start with `1` for all of these.